libpermian/plugins/anaconda_webui/__init__.py

```diff
@@ -0,0 +1,259 @@
+import tempfile
+import subprocess
+import os
+import shutil
+import json
+import time
+import logging
+import threading
+import urllib.request
+import urllib.error
+
+from libpermian.plugins import api
+from libpermian.result import Result
+from libpermian.workflows.isolated import IsolatedWorkflow
+from libpermian.events.structures.base import BaseStructure
+from libpermian.plugins.compose.exceptions import ComposeNotAvailable
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+@api.events.register_structure('AnacondaIS')
```

Comment:
I'm not a big fan of of such a event structure. The issue with this specific one is that it's single architecture only and also don't take into account possible additional repositories such as AppStream which are needed to be used on RHEL along with BaseOS.